elasticsearch/src/main/java/org/hibernate/search/elasticsearch/schema/impl/Elasticsearch2SchemaTranslator.java

```diff
@@ -503,7 +503,7 @@ else if ( indexedNullToken instanceof Boolean ) {
 			return new JsonPrimitive( (Boolean) indexedNullToken );
 		}
 		else {
-			throw LOG.unsupportedNullTokenType( mappingBuilder.getBeanClass(), fieldPath.getAbsoluteName(),
+			throw LOG.unsupportedNullTokenType( mappingBuilder.getBeanClass().getName(), fieldPath.getAbsoluteName(),
```

Comment:
oh.. you're right. I'm trying but it feels like I just found a new mining vein... let's see how far it goes :)